staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```
@@ -220,7 +223,13 @@ func (rules *ClientConfigLoadingRules) Load() (*clientcmdapi.Config, error) {
 	}
 
 	if rules.WarnIfAllMissing && len(missingList) > 0 && len(kubeconfigs) == 0 {
-		klog.Warningf("Config not found: %s", strings.Join(missingList, ", "))
+		missingString := strings.Join(missingList, ", ")
+
+		if rules.WarningHandler != nil {
+			rules.WarningHandler(fmt.Sprintf("Config not found: %s", missingString))
+		} else {
+			klog.Warningf("Config not found: %s", missingString)
```

Don't log, if `WarningHandler` is nil, just silently ignore the problem.

---

Alternatively, you can use `klog.V(1).Infof`.